iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/persistence/node/NodeInfo.java

```diff
@@ -224,6 +224,24 @@ public DataNodeConfigurationResp getDataNodeConfiguration(
     return result;
   }
 
+  /** Return the number of registered Nodes. */
+  public int getRegisteredNodeCount() {
+    int result;
+    dataNodeInfoReadWriteLock.readLock().lock();
+    try {
+      result = registeredDataNodes.size();
+    } finally {
+      dataNodeInfoReadWriteLock.readLock().unlock();
+    }
+    configNodeInfoReadWriteLock.readLock().lock();
+    try {
+      result += registeredConfigNodes.size();
```

Comment:
do we need to add confignode size?

Reply:
Yes, this makes the statistics of scatter width easier. Otherwise, we should take another map step when processing DataNodeId.

Reply:
got it~